delft/packet-importer.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
+let
+  importer = pkgs.callPackage ../hydra-packet-importer { };
+in
+{
+  deployment.keys."hydra-packet-import.json" = {
+    keyFile = ../hydra-packet-import.json;
+    user = "hydra-packet";
+  };
+
+  users.extraUsers.hydra-packet =
+    { description = "Hydra Packet Machine Importer";
+      group = "hydra";
+    };
+
+  system.activationScripts.hydra-packet = lib.stringAfter [ "users" ]
```

Comment:
you should be able to replace the whole activation script by

systemd.tmpfiles.rules = [
  "d /var/lib/hydra/packet-import 0755 hydra-packet hydra -"
  "f /var/lib/hydra/packet-import/machines 0644 hydra-packet hydra -"
];
```

Reply:
Nice, thanks!